### DISCUSSION TITLE
how to make a recipe book?

### DISCUSSION THREAD

**Discussion by Timofffee** [Minestom/Minestom#1609] [Help]
hello. I'm wondering how you can interact with the recipe book that is in the player's inventory. how can I do this? I looked at the events that are in the javadoc list, but I still didn't understand how best to implement it.

**Comment by Parul240** [comment]
I didn't understand 🤔

**Reply by sirBatiste** [reply]
maybe he want to wnow hot to add/remove items to the recipe book inventory, mee too
